mock/py/mockbuild/package_manager.py

```diff
@@ -496,6 +505,15 @@ def __init__(self, config, buildroot, plugins, bootstrap_buildroot, is_bootstrap
         if bootstrap_buildroot is None:
             self._check_command()
 
+    def initialize_vars(self):
+        self.buildroot.root_log.debug('configure yum vars')
+        pkg_manager_vars = self.get_pkg_manager_vars()
+        var_path = self.buildroot.make_chroot_path('etc/yum/vars/')
+        if pkg_manager_vars:
+            for key in pkg_manager_vars.keys():
+                with open(os.path.join(var_path, key), 'w+') as conf_file:
+                    conf_file.write(pkg_manager_vars[key])
```

Comment:
I think this could be deduplicated, and moved to the parent class.  The decision
where to store the variables can be done per `package_manager` content option.

Reply:
Ping on this as well.